src/bindings.cc

```diff
@@ -216,6 +208,29 @@ Napi::Value oboeInit(const Napi::CallbackInfo& info) {
     return Napi::Number::New(env, result);
 }
 
+//
+// Check to see if oboe is ready to issue sampling decisions.
+//
+// returns coded status as below
+//
+Napi::Value isReadyToSample(const Napi::CallbackInfo& info) {
+  int ms = 0;  // milliseconds to wait
+  if (info[0].IsNumber()) {
+    ms = info[0].As<Napi::Number>().Int64Value();
+  }
+
+  int status;
+  status = oboe_is_ready(ms);
+
+  // UNKNOWN 0
+  // OK 1
+  // TRY_LATER 2
+  // LIMIT_EXCEEDED 3
+  // INVALID_API_KEY 4
```

Comment:
Oboe won't return `INVALID_API_KEY` anymore because the collector will go into soft-disable mode instead of returning this error.

Reply:
so 4 is unused and the others remain the same, right?

Reply:
correct